pyvisa-py/highlevel.py

```diff
@@ -93,25 +93,53 @@ def get_debug_info():
     def _init(self):
 
         #: map session handle to session object.
-        #: dict[int, session.Session]
+        #: dict[int, sessions.Session]
         self.sessions = {}
 
-    def _register(self, obj):
+        #: map event handle to event object.
+        #: dict[int, Event]
+        self.events = {}
+
+    def _generate_handle(self):
+        """Creates a random but unique handle for a session object,
+        event or find list
```

Comment:
What do you mean by `find list` ?

Reply:
A find list is a VISA object used to iterate over available resources: http://zone.ni.com/reference/en-XX/help/370131S-01/ni-visa/vifindrsrc/ As far as I understand the documentation, find list handles share the pool together with handles of resource managers, resource sessions, and events. A session can't have the same handle as an event or a find list because eg. viClose() needs to work for all of these object types.

Reply:
Ok in that case I suggest the following:
```python
''''Creates a random and unique handle.

Handles are used for:
- session object (viSession)
- events (viEvent)
- find list (viFindList)
""" 

This way it is easier to find the reference when looking. My confusion was also coming from the fact that pyvisa never expose find lists.